### DISCUSSION TITLE
Problems with ..._step on GLEDOPTO

### DISCUSSION THREAD

**Discussion by MacValley** [Koenkk/zigbee2mqtt#7459] [Devices]
I am using a Gledopto GL-C-008-1ID to control a RGB-strip. Controlled via MQTT-messages. My problems concern controlling the color temp with step or move. To control **brightness** with move/stop or step works brilliant, e.g. with:
mosquitto_pub -h 192.168.1.169 -m '{"**brightness_step**": -20}' -t "zigbee2mqtt_h169/u69gled/set".
But when I try to change the color temp nothing happens: 
mosquitto_pub -h 192.168.1.169 -m '{"**color_temp_step**": -20}' -t "zigbee2mqtt_h169/u69gled/set".
Neither - nor positive values have any effect. Also color_temp_move has no effect.
Has anyone else tried this with success? Some flaw on the Gledopto GL-C-008-1ID (I have no idea on how to update firmware on that device)? 
I use the latest zigbee2mqtt (docker), everything is updated.

**Comment by Koenkk** [comment]
Looks that the gledopto devices don't support `color_temp_step`. These devices also don't support firmware updates so this won't work for this device.
